src/util/util_bit.h

```diff
@@ -156,7 +156,11 @@ namespace dxvk::bit {
   }
 
   inline uint32_t lzcnt(uint32_t n) {
-    #if (defined(_MSC_VER) && !defined(__clang__)) || defined(__LZCNT__)
+    #if defined(_MSC_VER) && !defined(__AVX2__)
```

Comment:
why AVX2?

Reply:
Bases on MSVC document, MSVC only predefine AVXs, there is no predefine for BMI or LZCNT. That's why I have to use AVX2 predefine to check it as all AVX2 processors support LZCNT. https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

Reply:
The problem I have with this is that building DXVK with AVX2 is inherently unsafe, and locking lzcnt behind it will lock out non-msvc out of using the fast path too even when BMI and/or SSE4a (not 4.1) are enabled.

Reply:
Fixed, now MSVC will always use BSR, until they add support for LZCNT flags.
I have also raised a ticket for this in Microsoft community: https://developercommunity.visualstudio.com/t/Lacks-LZCNT-predefined-flag-and-LZCNT-wi/10882435